artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/RoutingContextImpl.java

```diff
@@ -128,6 +133,29 @@ public void addQueue(final SimpleString address, final Queue queue) {
       queueCount++;
    }
 
+   @Override
+   public String debug() {
```

Comment:
this looks very much like a toString method

Reply:
I want to keep it separate..  and call it explicitly from trace methods only.

Reply:
solved